media/js/app/assetmgr/collection.js

```diff
@@ -482,7 +482,7 @@ CollectionList.prototype.updateSwitcher = function () {
     Mustache.update("switcher_collection_chooser", self.switcher_context, { parent: self.parent });
 
     // hook up switcher choice owner behavior
-    jQuery(self.el).find("a.switcher-choice.owner").unbind('click').click(function (evt) {
+    jQuery(self.el).find("a.switcher-choice.owner").off('click').on('click', function (evt) {
```

Comment:
When I switched this to a delegate pattern, e.g.:

```js
jQuery(self.el).on('click', 'a.switcher-choice.owner', function (evt) {
```
there were some lettuce test failures. I switched it back to `.off().on()` for now, because in this state it's working. I'm okay with refactoring this event listener to a delegate pattern later, but if you think it should be addressed now I'm also fine with looking deeper into the problem.